lib/config/virtual_container_host.go

```diff
@@ -81,6 +81,9 @@ type VirtualContainerHostConfigSpec struct {
 
 	// configuration for vic-machine
 	CreateBridgeNetwork bool `vic:"0.1" scope:"read-only" key:"create_bridge_network"`
+
+	// vic-machine create options used to create or reconfigure the VCH
+	VicMachineCreateOptions []string `vic:"0.1" scope:"read-only" key:"vic_machine_create_options"`
```

Comment:
I can do that but are we sure we want to persist the user/pass in the first place? I mean, what benefit does it have if nobody can see it anyway because we obscure it?